resources/js/components/modals/AdminShowConversation.vue

```diff
@@ -0,0 +1,39 @@
+<template>
```

Comment:
I prefer to have these components together, so if you would be so kind, please change the location to components/tabs/admin/components (the last components folder you would have to make if you haven't already). And then change the name to ShowConversationModal

Reply:
done